advancedTodo/src/redux/todoAppComplete/todoApp.ui.js

```diff
@@ -77,100 +78,100 @@ class TodoApp extends Component {
 	}
 
 	createNotification(type) {
-    switch (type) {
-      case 'success':
-        NotificationManager.success('Information successfully saved', 'Congratulations !');
-        break;
-      case 'deleteSuccessful':
-        NotificationManager.success('Todo Deleted Successfully', 'Congratulations !');
-        break;
-      case 'completeSuccessful':
-        NotificationManager.success('Todo Completed Successfully', 'Congratulations !');
-        break;
-      case 'error':
-        NotificationManager.error('Please add information to save', 'Warning', 5000, () => {
-          alert('Error');
-        });
-        break;
-			default :
-		   	return undefined
-    }
-  }
-
- handleSearch(e){
- 	let {newTodos} = this.props;
- 	let currentList = newTodos.filter( el => (el.name).includes(e.target.value));
- 	this.setState({
- 		currentList
- 	})
- }
-
- onFocus(e){
- 	this.setState({
- 		searchActive: true
- 	})
- }
-
- handleKeyPress(e){
- 	if(e.key === 'Enter'){
- 		this.handleOnClick()
- 	}
- }
+		switch (type) {
+			case 'success':

```

Comment:
Define constants for the hardcoded strings like 'Congratulations!', 'Information successfully saved', etc.  This will improve code readability and maintainability.